test/unit/controllers/components/omnibar.js

```diff
@@ -67,21 +67,6 @@ describe("OmnibarController", () => {
 
         });
 
-        it('Dispatches "ERROR" when student is not found',()=>{
```

Comment:
This test no longer makes sense. On a student not found the rfid workflow will trigger.